v1.5.0.md

```diff
@@ -0,0 +1,337 @@
+
+## Highlights
+
+### Experimental Features
+
+#### Longhorn V2 Data Engine Enhancements
+
+Harvester v1.5.0, which uses Longhorn v1.8.1, supports the creation of virtual machine images and boot volumes using the V2 Data Engine, and live migration of virtual machines with attached V2 volumes. To leverage the enhancements, you must create a new V2 StorageClass and then use it to create images and volumes after upgrading to Harvester v1.5.0. V2 volumes created using Harvester v1.4.x remain non-migratable after the upgrade.
+
+[Demo](https://www.youtube.com/watch?v=cRE7xR83uWo) | [Documentation](https://docs.harvesterhci.io/v1.5/advanced/longhorn-v2/) | GitHub issues: [#7521](https://github.com/harvester/harvester/issues/7521) and [#7522](https://github.com/harvester/harvester/issues/7522)
+
+### Fully Supported Features
+
+#### Third-Party Storage Support
+
+Harvester now supports provisioning of root volumes and data volumes using external container storage interface (CSI) drivers. This enhancement allows you to select drivers that meet specific requirements, such as performance optimization or seamless integration with existing storage solutions.
+
+Once the CSI driver is installed and the Harvester cluster is configured, an external storage solution can be used to store and manage virtual machine images, store virtual machine disks on volumes provisioned by the CSI driver, and perform other critical storage functions.
+
+[Demo](https://www.youtube.com/watch?v=tDK3lmKU9qI) | [Documentation](https://docs.harvesterhci.io/v1.5/advanced/csidriver) | GitHub issues: [#1199](https://github.com/harvester/harvester/issues/1199) and [#2272](https://github.com/harvester/harvester/issues/2272)
+
+#### ARM64 Support
+
+Harvester v1.5.0 fully supports deployment on hosts with ARM architecture. This includes add-ons, which were previously packaged only for x86_64-based hosts, and upgrade paths for ARM-only clusters. Harvester on ARM is fully tested, stable, and ready for mission-critical workloads.
+
+[Documentation](https://docs.harvesterhci.io/v1.5/install/requirements#hardware-requirements) | GitHub issue: [#6257](https://github.com/harvester/harvester/issues/6257)
+
+## Features
+
+- [GUI] [FEATURE] Consume external/third-party storage within Harvester [#7572](https://github.com/harvester/harvester/issues/7572)
+- [FEATURE][UI] Add EFI persisent state checkbox in VM advanced tab [#7557](https://github.com/harvester/harvester/issues/7557)
+- [FEATURE] Longhorn v2 data engine - live migration [#7522](https://github.com/harvester/harvester/issues/7522)
+- [FEATURE] Longhorn v2 data engine - boot volume support [#7521](https://github.com/harvester/harvester/issues/7521)
+- [GUI] [FEATURE] Add support for persistent TPM in VMs [#7187](https://github.com/harvester/harvester/issues/7187)
```

Comment:
It seems some issue titles having multiple area brackets are somehow inconsistent. This should be improved later.

Reply:
Do you mean we need to have a convention about issue title?

Besides, I'm wondering if we should group those issues together, since they're similar based on their titles.

```
- [FEATURE] Add support for persistent TPM in VMs [#6731 (https://github.com/harvester/harvester/issues/6731)
- [GUI] [FEATURE] Add support for persistent TPM in VMs [#7187 (https://github.com/harvester/harvester/issues/7187)
```